website/screens/components/header/code/HeaderCodePage.tsx

```diff
@@ -120,12 +120,12 @@ const sections = [
         content: <Example example={basicUsage} defaultIsVisible />,
       },
       {
-        title: "With custom content",
-        content: <Example example={withCustomContent} defaultIsVisible />,
+        title: "Custom content",
+        content: <Example example={customContent} defaultIsVisible />,
       },
       {
-        title: "With dropdown",
-        content: <Example example={withDropdown} defaultIsVisible />,
+        title: "Dropdown Header",
```

Comment:
Change Header to header.

Reply:
It refers to Header component and we are putting the components in upper case, so to be consistent here I would keep it as Header.

Reply:
So in that case we should check the other references to the header in this page? Because they are all in lower case. It is true that they can have a second meaning, referring to our component or just a normal header... How do you see it?

Reply:
For me it is the same case as here: 
![image](https://user-images.githubusercontent.com/10975289/174790919-218cda96-e647-466c-ac7f-5f19ae7afb49.png)

We are specifying that the Accordion is the Accordion component and that's why it is in upper case. In this case we want to say that it is an example of the Dropdown of the Header. Maybe we could change it to `Dropdown` only?

Reply:
I would put `Dropdown` only, as we put `Icons` for the examples with icons. Here if we put `Dropdown`, we are saying that it is an example with a dropdown. If we put `Dropdown child` maybe it could be doubtful.

Reply:
Okay 👍👍

Reply:
Done!